kolibri/core/content/api.py

```diff
@@ -227,6 +227,22 @@ def map_file(file):
     return file
 
 
+kind_activity_map = {
```

Comment:
Is this the final backend definition? If yes, we'll need to change [frontend constants](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/constants.js#L32) to lower case. I can open a PR for that.

Reply:
All of our other constants for content are lower case, so this seemed consistent.

Reply:
Also, if we could change the name of the map from `LearningActivityKinds` to just `LearningActivities` it would sit better with me - want to avoid confusing them with the `ContentNodeKinds`.

Reply:
https://github.com/learningequality/kolibri/pull/8180